canonical/ubuntu.com
# [WD-14301] use an issue form to report issues on GitHub

## Done

- An *issue form* is a new feature provided by GitHub that allows to render a form with fields and validations when creating a new issue on a repository. More info can be found in the official docs here: https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository#creating-issue-forms
- Also fixed the link from "Report a bug on this site" in the footer to use the new issue form (previously it wasn't using any template at all).

## QA

- Unfortunately, since it's a new issue template (called ISSUE_TEMPLATE.yaml) that doesn't exist on production yet, you won't be able to QA it. It needs the template to already be in the repository's main branch. But currently there is an issue form that is called ALTERNATIVE_ISSUE_TEMPLATE that you can check out by manually creating an issue on the repository to see how the issue form is going to look like:
     - Go to Issues tab
     - Click on "New issue"
     - Choose "Alternative issue template" and click "Get started"
     -  See the new form template and play around with it. You will notice required fields and helper texts. Also "Reported from" field at the bottom that will show "GitHub" if you are reporting it manually from the repository, otherwise will show a link to a webpage from where the issue was reported (cannot be tested until merged).

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-14301

## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)
